Patch tests of arb-2.5 to be compatible with flint-2.4.4

Arb's testsuite, src/acb_modular/test/t-epsilon_arg.c, calls
fmpq_dedekind_sum. However, this function does not exist in
flint-2.4.4.

fmpq_dedekind_sum was introduced in
https://github.com/wbhart/flint2/commit/64f656fb8db591ea11db606c7905de9f47b05e84
and arith_dedekind_sum is now an alias to fmpq_dedekind_sum.

So I patch src/acb_modular/test/t-epsilon_arg.c to call
arith_dedekind_sum instead of fmpq_dedekind_sum.

See also http://trac.sagemath.org/ticket/17688#comment:8 .

diff -r -u src/acb_modular/test/t-epsilon_arg.c new/acb_modular/test/t-epsilon_arg.c
--- src/acb_modular/test/t-epsilon_arg.c	2015-01-28 19:20:07.000000000 +0200
+++ new/acb_modular/test/t-epsilon_arg.c	2015-02-11 14:07:30.947675579 +0200
@@ -49,7 +49,7 @@
         fmpz_mul_ui(fmpq_denref(arg), c, 12);
         fmpq_canonicalise(arg);
 
-        fmpq_dedekind_sum(t, d, c);
+        arith_dedekind_sum(t, d, c);
         fmpq_sub(arg, arg, t);
 
         fmpq_clear(t);
Only in new/acb_modular/test: t-epsilon_arg.c~
